src/lavatory/utils/artifactory.py

```diff
@@ -9,7 +9,8 @@
 # pylint: disable=redefined-builtin
 from requests.exceptions import (BaseHTTPError, ConnectionError, HTTPError, InvalidURL, RequestException)
 
-from ..credentials import load_credentials
+from ..credentials import load_credentials, load_slack_credentials
+from utils.slack import post_slack_message
```

Comment:
This file does not seem to be included in the PR.

Reply:
Sorry for the delay:
src/lavatory/utils/slack.py

I see in the PR?